src/reducers/maps.reducer.js

```diff
@@ -0,0 +1,24 @@
+import initialState from './initialState'
+import { MAPS_LOADED, MAP_REMOVED, MAP_CREATED } from '../actions/actionTypes'
+
+export default function maps(state = initialState.timeline, action) {
+  switch (action.type) {
+    case MAPS_LOADED:
+      return {
+        ...state,
+        maps: action.payload
+      }
+    case MAP_REMOVED:
+      return {
+        ...state,
+        maps: [...state.maps].filter(map => map != action.payload)
```

Comment:
use `!==` instead of `!=`

https://github.com/airbnb/javascript#comparison--eqeqeq

Reply:
also i'd imagine that `map === action.payload` will never be true because even if `map` and `action.payload` have the same _value_, the `===` does a pointer equality. you should be checking for the map's id